libopenage/util/strings.h

```diff
@@ -116,5 +116,10 @@ std::vector<std::string> split_escape(const std::string &txt,
                                       char delim,
                                       size_t size_hint = 0);
 
+/**
+ * Newline splitter that works with both \n and \r\n.
+ */
+std::vector<std::string> split_newline(const std::string &txt);
```

Comment:
could be returning std::string_views instead :)

Reply:
I tried, but it is creating so many problems with existing code that relies on `std::string` which also makes the code harder to read :/ 

For example, `std::stoul` won't accept a string view as an argument, so you have this confusing back and forth conversion between std::string_view and std::string in the renderer parsers...